### ISSUE TITLE
Crop list of site types and add explanation

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2WebSDL#28]
@aufdenkampe can verify, but I really don't think we're going to need to allow users to pick things like "volcanic vents."  Maybe we can come up with a shorter list and add a link to the descriptions on the ODM2 vocab pages.

**Comment by horsburgh** [comment]
OK - what is the short list?

**Comment by horsburgh** [comment]
@SRGDamia1 and @aufdenkampe - if you guys can send us the shorter list, we can modify what is loading for SiteType. I'll add this one to the next milestone if you can get us the list.

**Comment by horsburgh** [comment]
@SRGDamia1 and @aufdenkampe - I will add this issue to the next milestone if you can give me the short list of SiteTypes you would like to see. I can take a crack at this, but would be happy to hear from you.

**Comment by SRGDamia1** [comment]
A quick stab:
- wetland
- unknown?? (I hate giving people this option.  If they are installing a new sensor, they should know!)
- tidalStream
- Stream
- storm sewer
- spring
- shore
- pavement
- land
- lakeReservoirImpoundment
- laboratoryOrSamplePreparationArea
- house
- estuary
- atmosphere

**Comment by horsburgh** [comment]
@jcaraballo17 - I think this is a simple change in the back-end database. Assigning to you.

**Comment by aufdenkampe** [comment]
@SRGDamia1 , thanks for providing a first cut. Here's my modified list, based on http://vocabulary.odm2.org/sitetype/

- wetland
- unknown?? (I hate giving people this option.  If they are installing a new sensor, they should know!)
- tidalStream
- Stream
- storm sewer
- spring
- soilHole
- pavement
- land
- lakeReservoirImpoundment
- laboratoryOrSamplePreparationArea
- house
- estuary
- ditch
- aggregateGroundwaterUse (Is this really the only way to refer to a Well????? If so, this is a clear example of why we need to reconcile and refactor [ODM2 SiteTypes](http://vocabulary.odm2.org/sitetype/) with [ODM2 SamplingFeatureTypes](http://vocabulary.odm2.org/samplingfeaturetype/)).
- atmosphere

I removed the following for these reasons:
- shore. I'm concerned that users will select this for monitoring water bodies. 

I like Sara's idea to have a pop-up or link to the Definition.

Can we have "more options" at the bottom of the list, to allow the user to expand to the full list?

Needless to say, it would be good to sort this alphabetically. See https://github.com/ODM2/ODM2ControlledVocabularies/issues/28.

**Comment by SRGDamia1** [comment]
I thought "aggregateGroundwaterUse" was specifically not a well, but I don't actually know where a well does fall.

Votes on whether or not we should give users the option of "unknown":

- [ ] Sara - NO
- [ ] @aufdenkampe 
- [ ] @horsburgh 
- [ ] Arscott
- [ ] Shannon

**Comment by horsburgh** [comment]
Another option instead of "unknown" would be to add "other". That would just imply that it is something other than the things in the list - but may still be known.

**Comment by aufdenkampe** [comment]
@SRGDamia1, you are right that "aggregateGroundwaterUse" is not quite right. Here's the definition from http://vocabulary.odm2.org/sitetype/aggregateGroundwaterUse/, which came from USGS.
* An Aggregate Groundwater Withdrawal/Return site represents an aggregate of specific sites where groundwater is withdrawn or returned which is defined by a geographic area or some other common characteristic. An aggregate groundwater site type is used when it is not possible or practical to describe the specific sites as springs or as any type of well including 'multiple wells', or when water-use information is only available for the aggregate.
So it really doesn't fit, but there is nothing else even close to a well in http://vocabulary.odm2.org/sitetype/

Ideally we want "observationWell", but that is from the SamplingFeatures list (http://vocabulary.odm2.org/samplingfeaturetype/), and WOFpy (and many of our other apps?) expect that all SamplingFeatures are of Type=Site.

We have many examples of overlap and inconsistencies like this, which is why we need to revisit the topic of revising these two CVs, which has been discussed in the past but put to rest for a few years due to other priorities. See:
* https://github.com/ODM2/ODM2/issues/64
* https://github.com/ODM2/ODM2/issues/132
* [2015-10-09 BiGCZ Notes, 3. ODM2 Vocabularies that we need to revisit](https://docs.google.com/document/d/1Ok_lN37hdDXjD8H_ElOIRYbAHJ4_WPAPyxx0iiuSR8k/edit#bookmark=id.hmtel9nnfwn6)
* 2014-11-18 ODM2 Notes](https://docs.google.com/document/d/1P0y-NNNiKQ170dUkHmU2XUrHku2rVB9CNhFR3YVf6R8/edit#bookmark=id.kqpdm2u4f9dr)

I agree with Sara that we should not use "unknown". Jeff's suggestion of "other" implies that we would be adding "other" to the [SiteTypeCV](http://vocabulary.odm2.org/sitetype/). If we do that, why just add "well", in addition to "other".

Another suggestion might be to copy all the applicable SamplingFeatureTypes to also be SiteTypes, such as:
* borehole
* observationWell
* streamGage
* waterQualityStation
* weatherStation
This might be confusing to some, but could become part of our path toward resolving this persistent issue.  

Looping @emiliom into this conversation, as this pertains to ODM2.1

**Comment by horsburgh** [comment]
Need to finish this issue off. I don't think we need to solve the inconsistencies between SamplingFeatureTypeCV and SiteTypeCV to close this issue. 

@jcaraballo17 - can you please sort the list alphabetically?

@SRGDamia1 - right now the list contains neither "unknown" or "other" (see the sandbox). Is this what you want? I think users probably need an option for when none of the others fit.

@aufdenkampe - you suggested an extended list and providing definitions. Do we need to do those to close this issue?

**Comment by aufdenkampe** [comment]
@jcaraballo17 - the list needs to still be alphabetized

@horsburgh & @SRGDamia1 -- I think we should add "other", which was a great suggestion from Jeff. I would prefer to NOT add "unknown". Other than that, the presently cropped list looks to be a good match for my suggestion in https://github.com/ODM2/ODM2WebSDL/issues/28#issuecomment-322578447.

Once these two tasks are done, I think we can close this issue. However, it might be worth adding follow-up issues on these topics:
* Provide SiteType explanations from "Definitions" given by http://vocabulary.odm2.org/sitetype/. This will require a User Interface enhancement of some sort, that either provides these as text that is visible when the user hovers, or as a link to the record at http://vocabulary.odm2.org/sitetype/.
*  Resolve inconsistencies between SamplingFeatureTypeCV & SiteTypeCV #142
  * @horsburgh, I'm very interested in resolving this before the BiG CZ workshop. Would you be open to taking a first step toward that, by adding these terms to the Site Type list currently on EnviroDIY?
    * borehole
    * observationWell
    * streamGage
    * waterQualityStation
    * weatherStation
  * If yes, then we would want to remove these Site Types from our list:
    * aggregateGroundwaterUse (and anything with this would become "observationWell")

**Comment by horsburgh** [comment]
@aufdenkampe - let's keep these issues separate. If you are satisfied with closing the original issue once @jcaraballo17 has the list alphabetized and adds "Other" to the list, then let's close this issue. 

New issues should be handled in new issues. Can you draft these issues? Resolving the inconsistencies in SamplingFeatureTypeCV and SiteTypeCV is an ODM2 issue and is not required to close this issue. If we need to add these terms as SiteTypes to the envirodiy instance, we can do it. In ODM2 we never anticipated forcing people to use specific vocabularies. If we need those terms as SiteTypes for this use case, we should just add them to the underlying database.

**Comment by aufdenkampe** [comment]
@horsburgh, I agree. Let's keep these issues separate. I'll draft up any follow up issues that are needed.

If you're open to having the EnviroDIY ODM2 instance use a different SiteTypeCV than is presently at http://vocabulary.odm2.org/sitetype/, then I'll draft a new list this weekend, so it will be ready for deployment next week. It will be an early draft version of how we might want to revamp the ODM2 [SamplingFeatureTypeCV](http://vocabulary.odm2.org/samplingfeaturetype/) & [SiteTypeCV](http://vocabulary.odm2.org/sitetype/) as described in https://github.com/ODM2/ODM2/issues/142.

**Comment by horsburgh** [comment]
@aufdenkampe - your proposal seems reasonable. We want to make sure the system is going to work well for the users.

**Comment by aufdenkampe** [comment]
@fryarludwig & @jcaraballo17, according to the comments on Sep. 27, 2017, @horsburgh and I agreed the the following two changes needed to be made before this issue was closed. From what I can tell, they have not been implemented:
* Add "Other" to list of SiteTypes.
* Alphabetize list

Note that I created #119 to track the other remaining work that was discussed above.

**Comment by horsburgh** [comment]
@jcaraballo17 - this seems like a 2 minute change that could finish this issue off and allow it to be closed. I see that you have alphabetized the sites on the sandbox for release 0.5, but can you please add "Other" to the list of site types? That's all that needs to be done to finish this issue off. Should we add this issue to the 0.5 release?

**Comment by aufdenkampe** [comment]
It would be great to do this before the release of 0.5, but don't let it delay the release.

**Comment by jcaraballo17** [comment]
Got it! looks like we're ready for the release.
